Library/ViewModels/CommentCellViewModelTests.swift

```diff
@@ -41,28 +46,65 @@ internal final class CommentCellViewModelTests: TestCase {
     self.postTime.assertValueCount(1, "The relative time of the comment is emitted.")
   }
 
+  func testOutputs_replyButtonIsHidden_IsBacker_True_IsLoggedIn_True() {
```

Comment:
`replyButtonIsNotHidden`

Reply:
Ah so in this case I'm following the existing convention where we list the name of the output use an underscore delimiter to signify what properties are being tested with it.